ast/spec.md

```diff
@@ -166,6 +168,18 @@ interface Identifier <: Expression, Pattern {
 
 An identifier. Note that an identifier may be an expression or a destructuring pattern.
 
+
+# PrivateName
+
+```js
+interface PrivateName <: Expression, Pattern {
+  type: "PrivateName";
+  name: string;
```

Comment:
I could have been clearer, my suggestion is

```
interface PrivateName <: Expression, Pattern {
  type: "PrivateName";
  name: Identifier;
}
```
I'm not suggesting a `private: boolean` flag on `Identifier`.

`Identifier` nodes in our spec are currently _every_ case of non-keyword, non-string literals. The same way it's not `name: string` on `FunctionDeclaration` names or `MemberExpression` properties.